### PR TITLE
Add instructions

### LINKED ISSUE #58: Add Instructions
Right now, it is not clear for the user how the website should be used. Instructions can be added under the website name on the left side of the home page.

### PR DESCRIPTION
**Pull request by nssafir**
Add instructions to the left side of the homepage.

### CODE REVIEW

File: src/main/webapp/styles/index.css [hunk 1]
```diff
@@ -23,6 +23,15 @@ body {
   font-weight: bolder;
 }
 
+#title {
+  text-align: center;
+  padding: 20px;
```
Comment (juanuribe28) [inline src/main/webapp/styles/index.css]: [Nit] Replace px with some relative unit (vw, vh, vmin, vmax) to adapt to different screen sizes.

File: src/main/webapp/styles/index.css [hunk 2]
```diff
@@ -23,6 +23,15 @@ body {
...
+}
+#instructions {
+  font-size: 30%;
+  padding: 20px;
```
Comment (juanuribe28) [inline src/main/webapp/styles/index.css]: Same with px

**Review by juanuribe28 — CHANGES_REQUESTED** [review]

File: src/main/webapp/styles/index.css [hunk 3]
```diff
@@ -23,6 +23,18 @@ body {
   font-weight: bolder;
 }
 
+#title {
+  font-size: 200%;
+  text-align: center;
+  padding: 4vw;
+  margin: 0px;
```
Comment (juanuribe28) [inline src/main/webapp/styles/index.css]: [Optional] When setting a property to 0 you don't need to include units.

File: src/main/webapp/styles/index.css [hunk 4]
```diff
@@ -23,6 +23,18 @@ body {
...
+}
+#instructions {
+  font-size: 40%;
+  padding: 4vw;
+  margin: 0px;
```
Comment (juanuribe28) [inline src/main/webapp/styles/index.css]: [Optional] same as above

**Review by juanuribe28 — APPROVED** [review]

File: src/main/webapp/index.html [hunk 5]
```diff
@@ -12,6 +12,14 @@
   <body>
     <header id="header" class="base">
       <p id="title">Engage</p>
+      <p id="instructions">
+        We help you engage more deeply with your social media feed. 
+        Copy and paste a post, or upload an image from you feed. 
+        Then, press submit, and you'll see a collection of books and community projects
+        about each important keyword in your feed. If you'd like to contact a local politician
+        about an issue, enable the use of location data and find your local representatives 
+        on the right side of the screen.
```
Comment (daperlmut) [inline src/main/webapp/index.html:21]: [Optional] I'd create a second paragraph starting with "If you'd like to contact..."

**Review by daperlmut — APPROVED** [review]

**Pull request merged**